common/lib/xmodule/xmodule/vertical_block.py

```diff
@@ -75,6 +76,11 @@ def student_view(self, context):
 
         return fragment
 
+    def get_unit_display_name(self):  # pylint: disable=missing-docstring
```

Comment:
@alisan617 the missing docstring warning is to remind you that all Python methods should have a docstring. See our Python guidelines:

http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python_guidelines.html#docstrings

Please add a docstring and remove the pylint declaration.